IFC4x3/Properties/n/NominalDepth_0HZnE0qUaHuO00025QrE$V/DocProperty.xml

```diff
@@ -7,3 +7,4 @@
 		</DocLocalization>
 	</Localization>
 </DocProperty>
+
```

Comment:
This empty line. Already seen this in Alex's commit. @jmirtsch we need to look into this.